addons/payment/models/account_invoice.py

```diff
@@ -88,7 +88,8 @@ def _create_payment_transaction(self, vals):
         return transaction
 
     def payment_action_capture(self):
-        self.authorized_transaction_ids.s2s_capture_transaction()
+        for transaction in self.authorized_transaction_ids:
+            transaction.action_capture()
```

Comment:
```suggestion
        self.authorized_transaction_ids.action_capture()
```
No need to iterate on the transaction, `action_capture` supports being called with 0, 1 or n records :).

Reply:
thanks !